File: JavaPrograms/src/basic/BubbleSort.java
package basic;

public class BubbleSort {

    public static void main(String[] args) {
        int arr[] = { 64, 34, 25, 12, 22, 11, 90 };

        int size = arr.length;

        int newArr[] = sortArrayUsingBubbleSort(arr, size);

        for (int i : newArr) {
            System.out.println(i);
        }
    }

    private static int[] sortArrayUsingBubbleSort(int[] arr, int size) {
        
        for (int i = 0; i < size; i++) {
            for (int j = 0; j < size - i - 1; j++) {
                if(arr[j] > arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;

                }
            }
        }

        return arr;
    }

    
}

File: JavaScriptPrograms/BubbleSort.js
const arr = [64, 34, 25, 12, 22, 11, 90];
const size = arr.length;

function bubbleSort(arr, size) {
    
    for (let i = 0; i < size; i++) {
        for (let j = 0; j < size-i-1; j++) {
            if(arr[j] > arr[j+1]){
                let temp = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
        
    }
    return arr;
}

const newArr = bubbleSort(arr, size);
newArr.forEach(element => {
    console.log(element);
});